src/main/java/io/split/android/client/impressions/HttpImpressionsSender.java

```diff
@@ -28,40 +29,31 @@ public HttpImpressionsSender(CloseableHttpClient client, String eventsEndpoint,
     }
 
     @Override
-    public void post(List<TestImpressions> impressions) {
+    public boolean post(List<TestImpressions> impressions) {
+        if (impressions == null || impressions.isEmpty()) {
+            return true;
+        }
 
         if (!Utils.isReachable(_eventsEndpoint)) {
             Timber.d("%s is NOT REACHABLE. Avoid trying to send impressions this time.", _eventsEndpoint.getHost());
-            return;
+            return false;
         }
 
         synchronized (this) {
-            String[] chunkNames = _storageManager.getAllChunkNames();
-            Timber.i("Posting %d Split impressions", chunkNames.length);
-
-            for (String chunkId :
-                    chunkNames) {
-                try {
-                    String json = _storageManager.readStringChunk(chunkId);
-
-                    if(json == null){
-                        _storageManager.chunkFailed(chunkId);
-                        continue;
-                    }
-
-                    StringEntity entity = new StringEntity(json, "UTF-8");
-                    entity.setContentType("application/json");
-
-                    if (post(entity)) {
-                        _storageManager.chunkSucceeded(chunkId);
-                    } else {
-                        _storageManager.chunkFailed(chunkId);
-                    }
-
-                } catch (UnsupportedEncodingException e) {
-                    Timber.e(e);
-                    _storageManager.chunkFailed(chunkId);
+            Timber.i("Posting %d Split impressions", impressions.size());
```

Comment:
this should be in debug mode. Info should not show this log to be consistent with the rest of the SDKs.

Reply:
done